app/controllers/comments_controller.rb

```diff
@@ -1,5 +1,6 @@
 class CommentsController < ApplicationController
   before_action :set_comment, only: %i[ show edit update destroy ]
+  
```

Comment:
Maybe not necessary, but consider adding
```ruby
before_action { authorize @comment }
```
not only to this controller, but all to follow DRY principles.

Reply:
done